specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json

```diff
@@ -559,13 +565,13 @@
               },
               "x-ms-share-next-allowed-provisioned-iops-downgrade-time": {
                 "x-ms-client-name": "NextAllowedProvisionedIopsDowngradeTime",
-                "description": "Return timestamp for provisioned IOPS following existing rules for provisioned storage GiB.",
+                "description": "Returns the current share next allowed provisioned iops downgrade time.",
```

Comment:
Similar change is missing for SetShareProperties API

Reply:
Fixed.